server/graphql/v2/query/RequiredBankInformationQuery.ts

```diff
@@ -0,0 +1,23 @@
+import { GraphQLInt, GraphQLNonNull } from 'graphql';
+
+import RequiredBankInformation from '../object/RequiredBankInformation';
+import models from '../../../models';
+
+const RequiredBankInformationQuery = {
+  type: RequiredBankInformation,
+  args: {
+    collectiveId: {
```

Comment:
We avoid exposing auto-increment ids in GraphQL API v2, alternatives: slug, hash-ids.